apps/gauzy/src/app/pages/users/users.component.ts

```
@@ -143,94 +138,182 @@ export class UsersComponent extends PaginationFilterBaseComponent implements OnI
+		// Set the component view name
 		this.viewComponentName = ComponentEnum.USERS;
+
+		// Listen for layout changes related to the 'Users' component
 		this.store
 			.componentLayout$(this.viewComponentName)
 			.pipe(
+				// Avoid emitting if the layout style has not changed
 				distinctUntilChange(),
+
+				// Update the data layout style based on the emitted layout
 				tap((componentLayout: ComponentLayoutStyleEnum) => (this.dataLayoutStyle = componentLayout)),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)